components/raftstore/src/store/fsm/store.rs

```diff
@@ -1964,6 +1965,16 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
                 .swap(false, Ordering::SeqCst),
         );
 
+        let mut query_stats = QueryStats::default();
+        let query_num = self
+            .ctx
+            .global_stat
+            .stat
+            .engine_total_query_written
+            .swap(0, Ordering::SeqCst);
+        query_stats.set_put(query_num);
+        stats.set_query_stats(query_stats);
```

Comment:
Why only report write qps in store heartbeat for the whole store?

Reply:
read qps will be added in `handle_store_heartbeat`